exercises/giovanni_disiena/ex02_oo_basics/src/Shape.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SHAPE_H
+#define SHAPE_H
+
+#include <memory>
+#include "ShapeBase.h"
+
+class Shape {
+public:
+  Shape(std::shared_ptr<ShapeBase>);
+  Shape(const Shape &);
+  Shape &operator=(const Shape &);
+  std::shared_ptr<ShapeBase> getBase() const;
+private:
+  std::shared_ptr<ShapeBase> m_shapeBasePointer;
+};
+
+#endif // SHAPE_H
```

Comment:
I am slightly confused as to the purpose of this class, it doesn't seem to be used anywhere.

Reply:
I have seen it has been removed from the `CMakeLists.txt` so I assume you were removing it? If so you need to remove the file as well.